ansible/roles/sshd/templates/etc/ssh/ldap_authorized_keys_bindpw.j2

```diff
@@ -2,4 +2,4 @@
  # Copyright (C) 2022 DebOps <https://debops.org/>
  # SPDX-License-Identifier: GPL-3.0-only
  #}
-{{- sshd__ldap_bindpw -}}
\ No newline at end of file
+{{- sshd__ldap_bindpw -}}
```

Comment:
New line in password file might be significant. I hope that the template will be correct here, might need to be tested in an LDAP environment to be sure.